src/core_plugins/kibana/public/discover/embeddable/search_embeddable.js

```diff
@@ -0,0 +1,122 @@
+import angular from 'angular';
+import { Embeddable } from 'ui/embeddable';
+import searchTemplate from './search_template.html';
+import * as columnActions from 'ui/doc_table/actions/columns';
+
+export class SearchEmbeddable extends Embeddable {
```

Comment:
You mean like make the entire search embeddable be a react/redux component instead of angular? Not without a lot of work.  Unless you are talking about something much smaller?